os/include/tinyara/reboot_reason.h

```diff
@@ -45,7 +45,7 @@ typedef enum {
 
 	REBOOT_NETWORK_WIFICORE_WATCHDOG   = 80, /* Wi-Fi Core Watchdog Reset */
 	REBOOT_NETWORK_WIFICORE_PANIC      = 81, /* Wi-Fi Core Panic */
-
+	REBOOT_BOARD_SPECIFIC0             = 249, /* IWDG reboot*/
```

Comment:
REBOOT_BOARD_SPECIFIC6 = 246, /* Board Specific Reboot Reason */
REBOOT_BOARD_SPECIFIC7 = 247, 
REBOOT_BOARD_SPECIFIC8 = 248, 
REBOOT_BOARD_SPECIFIC9 = 249,

To consider further expansion, Could you please change like this?

Reply:
changed